cpp/tests/CMakeLists.txt

```diff
@@ -105,10 +105,12 @@ ConfigureTest(SPATIAL_WINDOW_POINT_TEST
 ConfigureTest(HAVERSINE_TEST_EXP
     experimental/spatial/haversine_test.cu)
 
+ConfigureTest(POINT_DISTANCE_TEST_EXP
```

Comment:
This PR adds two test files 

1. `experimental/spatial/point_distance_test.cu`
2. `spatial/point_distance_test.cpp`

But we only have a test for `point_distance_test.cu`, making me think that `point_distance_test.cpp` needs to be deleted.

Reply:
The `cpp` test was meant to test cudf column API and `cu` tests header only API. We need both so added both to CMakeLists.